prog/vm/nexus.rb

```diff
@@ -334,6 +334,14 @@ def write_params_json
       amount: vm.cores
     )
 
+    BillingRecord.create_with_id(
+      project_id: project.id,
+      resource_id: vm.id,
+      resource_name: vm.name,
+      billing_rate_id: BillingRate.from_resource_properties("VmStorage", vm.family, vm.location)["id"],
+      amount: storage_volumes.sum { _1["size_gib"] }
```

Comment:
nit: 
```suggestion
      amount: vm.storage_size_gib
```

Reply:
This is not needed anymore since we are creating separate billing records for each volume